### PR TITLE
after talking with teams, only 1 emoji should ship

### PR DESCRIPTION
**Pull request by babineaum**

### CODE REVIEW

**Comment by ivanvoznyakovsky** [comment]
:+1:

**Comment by goldcaddy77** [comment]
:shaved_ice:

File: map_configs/dx_map_config.yml [hunk 1]
```diff
@@ -97,7 +97,7 @@ issue:
     - match:
         repo: "(dxng|user-interface)"
         owner: "dataxu"
-        message: ".*:(shipit|sheep|ship|shell|shaved_ice):.*"
+        message: ".*:(shipit):.*"
```
Comment (dx-pbuckley) [inline map_configs/dx_map_config.yml]: I think we just need ".*:shipit:.*, the () are for grouping the regex when there are multiple options but we're explicitly going down to one.

**Comment by ferrants** [comment]
I don't like that changes in our config is a change in this repo. We should move our config elsewhere

**Comment by dx-pbuckley** [comment]
@ferrants where to?

**Comment by ferrants** [comment]
@dx-pbuckley anywhere? dxansible? we don't need to expose or churn our config in this repo.

**Comment by babineaum** [comment]
ok added update @ferrants would we create another file in this repo or something outside of the scope of here? are you thinking this is "open" and we should keep our custom changes somewhere else?

**Comment by ferrants** [comment]
@babineaum that's exactly what i'm thinking, file in another repo.

**Comment by babineaum** [comment]
ok can we get this in today so we aren't auto merging stuff for the dxng team, then ill work on pulling it out into another repo?

**Comment by ferrants** [comment]
sure. :+1: just thinking for the future

**Comment by dx-pbuckley** [comment]
I'm fine with moving our configure somewhere else and changing the deploy to pull it in from there. But for now let's :sheep: this and make sure DXNG is still working.

**Comment by goldcaddy77** [comment]
Thanks for the quick turnaround all!

**Pull request merged**